src/Marketplace/Services/RecipientService.php

```diff
@@ -38,25 +43,27 @@ public function __construct()
 
         $this->mundipaggApi = new MundiAPIClient($secretKey, $password);
```

Comment:
A gente consegue mudar o nome dessa variável pra `$this->pagarmeApi` com facilidade?